run_bot.sh

```
@@ -90,34 +115,118 @@ process_config() {
+    local override_meeting_url=$2
+    local recording_mode=${RECORDING:-true}  # Par défaut true
+    local debug_mode=${DEBUG:-false}  # Debug mode avec VNC
+    local debug_logs=${DEBUG_LOGS:-false}  # Debug logs mode
+    
+    if [ ! -f "$config_file" ]; then
+        print_error "Configuration file '$config_file' not found"
+        print_info "Please create a JSON configuration file. See params.json for example format."
+        exit 1
+    fi
+    
+    local output_dir=$(create_output_dir)
```

Shellcheck (0.10.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)